src/features/query-bar/__tests__/FilterButton-test.js

```diff
@@ -11,166 +11,119 @@ describe('feature/query-bar/components/filter/FilterButton', () => {
-                        operator: LESS_THAN,
-                        values: [],
-                    },
-                ],
+                columnId: '1',
+                id: '4',
+                operator: EQUALS,
+                values: [],
             },
-        ].forEach(({ description, conditionId, conditions, value, expectedConditions }) => {
-            test(`${description}`, () => {
-                const wrapper = getWrapper({ columns });
-                wrapper.setState({
-                    conditions,
-                });
-                wrapper.instance().handleOperatorChange(conditionId, value);
-
-                expect(wrapper.state('conditions')).toEqual(expectedConditions);
+        ];
+        const expectedConditions = [{ columnId: '1', id: '4', operator: LESS_THAN, values: [] }];
+        test.each`
+            conditionId | value
+            ${'4'}      | ${LESS_THAN}
+        `('should set conditions with an object with operator', ({ conditionId, value }) => {
+            const wrapper = getWrapper({ columns });
+            wrapper.setState({
+                conditions,
             });
+            wrapper.instance().handleOperatorChange(conditionId, value);
+
+            expect(wrapper.state('conditions')).toEqual(expectedConditions);
         });
     });
 
     describe('handleValueChange()', () => {
-        [
+        const conditions = [
             {
-                description: 'should set conditions with an object with value',
-                index: 0,
-                conditionId: '5',
-                conditions: [
-                    {
-                        columnId: '1',
-                        id: '5',
-                        operator: EQUALS,
-                        values: [],
-                    },
-                ],
-                values: ['0'],
-                expectedConditions: [
-                    {
-                        columnId: '1',
-                        id: '5',
-                        operator: EQUALS,
-                        values: ['0'],
-                    },
-                ],
+                columnId: '1',
+                id: '5',
+                operator: EQUALS,
+                values: [],
             },
-        ].forEach(({ description, conditionId, conditions, values, expectedConditions }) => {
-            test(`${description}`, () => {
-                const wrapper = getWrapper({ columns });
-                wrapper.setState({
-                    conditions,
-                });
-                wrapper.instance().handleValueChange(conditionId, values);
-
-                expect(wrapper.state('conditions')).toEqual(expectedConditions);
+        ];
+        const expectedConditions = [{ columnId: '1', id: '5', operator: EQUALS, values: ['0'] }];
+        test.each`
+            conditionId | values
+            ${'5'}      | ${['0']}
+        `('should set conditions with an object with value', ({ conditionId, values }) => {
```

Comment:
Similar comment.

I assume the above unit test changes do not change functionality and only affect test.each and whitespace.

Reply:
Yep